docs/UserGuide.md

```diff
@@ -183,8 +189,9 @@ Action     | Format, Examples
 **Add**    | `add n/NAME p/PHONE i/INCOME e/EMAIL a/ADDRESS f/FAMILY t/TAG [r/REMARK]` <br> e.g. `add n/John Doe p/98765432 i/20000 e/johnd@example.com a/311, Clementi Ave 2, #02-25 f/4 r/Buyer t/Owes $1000.`
 **Delete (by name)** | `delete n/NAME`<br> e.g. `delete n/John`
 **Delete (by index)** | `delete INDEX`<br> e.g. `delete 3`
-**edit** | `edit INDEX [n/NAME] [p/PHONE] [i/INCOME] [e/EMAIL] [a/ADDRESS] [f/FAMILY] [t/TAG] [r/REMARK]` <br> e.g. `edit 2 n/Denzel i/100000`
-**filter** | `filter STRING`<br> e.g. `filter David`
-**list**   | `list`
-**help** | `help`
-**exit**   | `exit`
+**Edit** | `edit INDEX [n/NAME] [p/PHONE] [i/INCOME] [e/EMAIL] [a/ADDRESS] [f/FAMILY] [t/TAG] [r/REMARK]` <br> e.g. `edit 2 n/Denzel i/100000`
```

Comment:
once the birthday field is merged we need to edit this to include `b/BIRTHDAY` !